addon/templates/components/columns/base.hbs

```diff
@@ -2,7 +2,9 @@
   {{component column.component column=column table=table tableActions=tableActions extra=extra sortIcons=sortIcons}}
 {{else}}
   {{label}}
-  {{#if sortIconProperty}}
+  {{#if (and sortIcons.iconComponent sortIconProperty)}}
```

Comment:
Can't remember, if we talked about that. but this way the `iconComponent` would only get rendered when there is a [`sortIconProperty`](https://github.com/RedTn/ember-light-table/blob/7975553815eda87054da4f0ec85d248c9cdedc77/addon/components/columns/base.js#L82-L92), i.e. when either of `sortable` of `sorted` is `true`.

I don't have strong feelings either way, since I can't come up with a scenario where you would want to render an `iconComponent` when you don't have any icons to display. Just wanted to have mentioned it. :smile:

Reply:
I remember I ran into this issue in the demo where the avatar column was displaying the sort component when sortable: 'false'.
Because the iconComponent is being specified in the hbs file on the table, all the columns will utilize the iconComponent, so we need a check if the user doesn't want a column to be sortable.

Reply:
Yup, but some users might argue that they'd want their `iconComponent` to display a "not sortable" icon and that it's up to the user to handle the "not sortable" state instead of ELT doing that for the user.

For instance:

```hbs
{{#if sortIconProperty}}
  <i class="lt-sort-icon material-icons" style="height:0px;">{{get sortIcons sortIconProperty}}</i>
{{/if}}
```

Reply:
Understood, I see the use case. Because you have no strong feelings I guess I'll leave the base.hbs as is for now.